src/Resources/views/Invoice/show.html.twig

```diff
@@ -23,84 +24,126 @@
+                    <th colspan="7" class="right aligned">
+                        <strong>{{ 'sylius_invoicing_plugin.ui.taxes_total'|trans([], 'messages', invoice.localeCode) }}</strong>:
+                    </th>
+                    <th id="invoice-tax-total" class="right aligned" {{ sylius_test_html_attribute('invoice-taxes-total') }}>
+                        {{ '%0.2f'|format(invoice.taxesTotal/100) }}
+                    </th>
+                    <th id="invoice-taxes-total-currency-code" class="single line" {{ sylius_test_html_attribute('invoice-taxes-total-currency-code') }}>
+                        {{ invoice.currencyCode }}
+                    </th>
+                </tr>
+                <tr>
+                    <th colspan="7" class="right aligned">
                         <strong>{{ 'sylius.ui.total'|trans }}</strong>:
                     </th>
-                    <th id="invoice-total" class="right aligned">
-                        {{ money.format(invoice.total, invoice.currencyCode) }}
+                    <th id="invoice-total" class="right aligned" {{ sylius_test_html_attribute('invoice-total') }}>
+                        {{ '%0.2f'|format(invoice.total/100) }}
+                    </th>
+                    <th id="invoice-total-currency-code" class="single line" {{ sylius_test_html_attribute('invoice-total-currency-code') }}>
+                        {{ invoice.currencyCode }}
                     </th>
                 </tr>
+
+                {% if invoice.taxItems.count() > 0 %}
+                    <tr>
+                        <th colspan="7" class="right aligned">
+                            <strong>{{ 'sylius_invoicing_plugin.ui.tax_rate'|trans }}</strong>
+                        </th>
+                        <th class="right aligned">
+                            <strong>{{ 'sylius_invoicing_plugin.ui.tax_amount'|trans }}</strong>
+                        </th>
+                        <th class="left aligned">
+                            <strong>{{ 'sylius.ui.currency'|trans }}</strong>
+                        </th>
+                    </tr>
+                    {% for item in invoice.taxItems %}
+                        <tr class="tax-item" {{ sylius_test_html_attribute('invoice-tax-item') }}>
+                            <th colspan="7" class="right aligned">
+                                <strong {{ sylius_test_html_attribute('invoice-tax-item-label') }}>{{ item.label }}</strong>:
+                            </th>
+                            <th class="right aligned tax-item-amount" {{ sylius_test_html_attribute('invoice-tax-item-amount') }}>
+                                {{ '%0.2f'|format(item.amount/100) }}
+                            </th>
+                            <th class="single line" {{ sylius_test_html_attribute('invoice-tax-item-currency-code') }}>
+                                {{ invoice.currencyCode }}
+                            </th>
+                        </tr>
```

Comment:
Indenting HTML inside twig functions would result in not-so-well-looking code in a result file 🐎